docs/discussion.md

```diff
@@ -0,0 +1,232 @@
+# btwxt 
+## *Big Ladder Software*
```

Comment:
Maybe replace these lines with a single title that better describes the content of this document? The file name is "discussion.md", but I feel that isn't the most descriptive name. What if we call the file "methodology.md", and make the title "btwxt Methodology"?